modules/test/playwright/tests/login-web/main/login.spec.ts

```diff
@@ -105,3 +106,40 @@ test('LPD-28406 Access to My Account is allowed after disabling access to it thr
 		);
 	}
 });
+
+test('LPD-55426 Test sign in button is disabled until page is fully loaded', async ({
+	context,
+	page,
+}) => {
+	await performLogout(page);
+
+	const cdpSession = await context.newCDPSession(page);
+	await cdpSession.send('Network.emulateNetworkConditions', {
+		connectionType: 'cellular3g',
+		downloadThroughput: (750 * 1024) / 8,
+		latency: 70,
+		offline: false,
+		uploadThroughput: (250 * 1024) / 8,
+	});
+
+	await page.goto(liferayConfig.environment.baseUrl + '/c/portal/login', {
+		waitUntil: 'commit',
+	});
+	await expect(page.getByRole('button', {name: 'Sign In'})).toHaveAttribute(
+		'disabled'
+	);
+	await page.waitForLoadState('domcontentloaded');
+	await expect(page.getByRole('button', {name: 'Sign In'})).toBeEnabled();
+
+	await page.goto(liferayConfig.environment.baseUrl, {
+		waitUntil: 'domcontentloaded',
+	});
+	await page.getByRole('button', {name: 'Sign In'}).click();
+	await expect(page.getByText('Forgot Password')).toBeVisible();
+	await expect(
+		page.getByRole('button', {name: 'Sign In'}).last()
+	).toHaveAttribute('disabled');
```

Comment:
I don't get this. It should not be enabled? (the next expect is to verified that is toBeEnabled)

Reply:
I made that change after several tests with playwright
I see that if the page.goto use the default waitUntil that is "load", when the sign in button is clicked, the modal appears fully loaded. If I used "commi", the modal is not shown
After changing the waitUntil to "domcontentloaded" and clicking in sign in, it shows the full loading of the modal, first with the button disabled and then enabling it.

Reply:
https://testray.liferay.com/#/project/35392/routines/45357/build/216882985/case-result/216883241